files/rules.yaml

```diff
@@ -19,3 +19,12 @@ egress:
     protocol: UDP
     to:
       - 0.0.0.0/0
+ingress:
```

Comment:
It is not possible to define ingress to the firewall this way. The firewall rules are only ment to define rules in the forwarding path, either ingress or egress from a machine perspective, not from the firewall perspective itself

Reply:
What do you mean with a input rule?